test/functional/cmdline_options_tester/src/TestConfigParser.java

```diff
@@ -191,6 +191,28 @@ public void xmlStartElement(String elementName, Hashtable attributes) {
 					_currentOutput = null;
 				}
 
+			} else if (elementName.equalsIgnoreCase("saveoutput")) {
+				if (hasAllowedPlatform((String)attributes.get("platforms"))) {
+					String regex = (String)attributes.get("regex");
+					String javaUtilPattern = "no";
+					String showRegexMatch = "no";
+					if (null != attributes.get("showMatch")) {
+						showRegexMatch = (String)attributes.get("showMatch");
+					}
+					String caseSensitive = (String)attributes.get("caseSensitive");
+					String saveName = (String)attributes.get("saveName");
+					try {
+						int splitIndex = Integer.parseInt((String) attributes.get("splitIndex"));
+					} catch (NumberFormatException e) {
+						System.err.println("String split failure: splitIndex is not a number");
```

Comment:
actually, I think the better approach would be to keep splitIndex as a string here, and throw all the errors in the match code. This way it is consistent.

Reply:
I put it in the constructor of `saveoutput.java`